tests/extensions/test_pointcloud.py

```diff
@@ -0,0 +1,102 @@
+import json
+import unittest
+# from copy import deepcopy
+
+import pystac
+from pystac import (Item, Extensions)
+from pystac.extensions import ExtensionError
+from tests.utils import (TestCases, test_to_from_dict)
+
+
+class PointcloudTest(unittest.TestCase):
```

Comment:
Missing property tests for density and statistics.

Reply:
added